3-reveal/migrations/5-IRS/4-Zambia-2020/deploy/zambia_daily_summary_event.psql

```diff
@@ -0,0 +1,41 @@
+-- Deploy reveal_irs_zambia_2020:zambia_daily_summary_event to pg
+-- requires: reveal_transaction_tables:events
+
+BEGIN;
+
+SET search_path TO :"schema",public;
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS zambia_daily_summary_event AS
+SELECT 
+  id,
+  base_entity_id,
+  location_id,
+  event_type,
+  provider_id,
+  date_created,
+  event_date,
+  entity_type,
+  task_id,
+  team_id,
+  events.server_version,
+  tasks.plan_identifier as plan_id,
+  events.created_at,
+  (form_data->>'bottles_full')::int AS bottles_full,
+  (form_data->>'bottles_accounted')::int AS bottles_accounted,
+  (form_data->>'bottles_empty')::int AS bottles_empty,
+  (form_data->>'bottles_lostdamaged')::int AS bottles_lostdamaged,
+  (form_data->>'bottles_start')::int AS bottles_start,
+  (form_data->>'found')::int AS daily_found,
+  (form_data->>'sprayed')::int AS daily_sprayed
+FROM events
+INNER JOIN tasks ON tasks.identifier = events.task_id
```

Comment:
It was feeling tasks without this, but now I think something must be wrong with the tests, like this migration not running, since `plan_id` should be here based on that, https://github.com/OpenSRP/opensrp-reveal-datawarehouse/blob/master/3-reveal/migrations/2-transaction_tables/deploy/more_event_fields.psql

To get passing looks like we must to fix tests, then revert the join and select changes to get this passing

Reply:
Note: the reason i didn't use a join here is because most if not all daily summary events don't have have task_id as null

Reply:
yea gotcha, I added a join b/c this was failing integration tests w/ `plan_id` does not exist in this table, but then like the comment above states, I noticed it's added to the table in a migration that appears not to be run before the tests run (I guess?) hence the error

Reply:
definitely don't want to do a join unless we want to

Reply:
@pld i don't think is fully resolved we either need to use the plan_id or have have values on the task_id field

Reply:
yea definitely prefer plan_id so we can remove the join